src/components/main-page/Partners/Partners.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable @next/next/no-img-element */
+import { Badge, Heading, Section } from "components/ui";
+import { useTranslation } from "next-i18next";
+import styles from "./Partners.module.scss";
+
+type CompanyInfo = {
+  name: string;
+  url?: string;
+  logo: string;
+};
+
+interface Partners {
+  name: string;
+  companies: CompanyInfo[];
+}
+
+const Partners = () => {
+  const { t } = useTranslation();
+  const partners = t("partners.categories", {
+    returnObjects: true,
+  }) as Partners[];
+
+  return (
+    <Section variant="white" id="partners" className={styles.section}>
+      <Heading className={styles.heading} withAccent>
+        {t("partners.title")}
+      </Heading>
+      <div className={styles.container}>
+        {partners.map(({ name, companies }) => {
+          return (
```

Comment:
We don't need `return` here - but it's minor thing - if you think that it's more readable with `return` you can leave it as it is 😀

Reply:
Yeah, I really like this explicit return, so I'd like to leave it as it is 😁